src/custom/meteo-france/components/MapComponent.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div class="map-wrapper">
+    <div ref="mapContainer" class="map-container"></div>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
Is there a reason to use a script with defineComponent here instead of the script setup like below ?

Reply:
thanks